.travis/install.sh

```diff
@@ -3,43 +3,43 @@ set -v
 
 export COMMIT_MSG=$(git show HEAD^2 -s)
 export PULP_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulpcore\/pull\/(\d+)' | awk -F'/' '{print $7}')
-export PULP_PLUGIN_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulpcore-plugin\/pull\/(\d+)' | awk -F'/' '{print $7}')
 export PULP_SMASH_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/PulpQE\/pulp-smash\/pull\/(\d+)' | awk -F'/' '{print $7}')
 export PULP_FILE_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp_file\/pull\/(\d+)' | awk -F'/' '{print $7}')
+export PULP_CERTGUARD_PR_NUMBER=$(echo $COMMIT_MSG | grep -oP 'Required\ PR:\ https\:\/\/github\.com\/pulp\/pulp-certguard\/pull\/(\d+)' | awk -F'/' '{print $7}')
 
 
-cd .. && git clone https://github.com/pulp/pulpcore.git
+cd .. && git clone --depth=1 https://github.com/pulp/pulpcore.git --branch master
 
 if [ -n "$PULP_PR_NUMBER" ]; then
   pushd pulpcore
-  git fetch origin +refs/pull/$PULP_PR_NUMBER/merge
-  git checkout FETCH_HEAD
+  git fetch --depth=1 origin pull/$PULP_PR_NUMBER/head:$PULP_PR_NUMBER
+  git checkout $PULP_PR_NUMBER
   popd
 fi
 
 pip install -e ./pulpcore[postgres]
+sed -i "s/pulp:80/localhost:24817/g" ./pulpcore/.travis/test_bindings.rb
```

Comment:
I can see immediately where this comes from (my bad).
But should we not strive for consistency here, and start the same ci-container as used in pulpcore?
Or since we only want to test the bindings-generation, we could use the ready to use container from https://pulp.plan.io/issues/6205.

Reply:
I'm not sure, for one side I believe it will make CI slower, for the other side we will have consistency and we rarely touch this repo

Reply:
And we'd have more incentive to make that other method faster...
But that's probably not the best reason to do it.
We could defer that question into a new issue and say: This here is fixing the CI today, so take it.